packages/mdc-drawer/_mixins.scss

```diff
@@ -14,98 +14,89 @@
 // limitations under the License.
 //
 
-@import "@material/animation/functions";
-@import "@material/animation/variables";
 @import "@material/theme/mixins";
-@import "@material/theme/variables";
-@import "@material/typography/mixins";
-
-@mixin mdc-drawer-toolbar-spacer_() {
-  .mdc-drawer__toolbar-spacer {
-    display: flex;
-    position: relative;
-    flex-direction: row;
-    flex-shrink: 0;
-    align-items: center;
-    box-sizing: border-box;
-    height: 56px;
-    padding: 16px;
-    border-bottom: 1px solid rgba(0, 0, 0, .12);
-
-    // TODO(sgomes): replace with global breakpoints when we have them
-    @media (min-width: 600px) {
-      height: 64px;
-    }
+@import "@material/list/mixins";
+@import "@material/ripple/mixins";
+
+@mixin mdc-drawer-active-overlay-color($color) {
```

Comment:
Suggestions:

* Rename this `activated-overlay-color`
* Remove `mdc-states-selected`, since IIUC drawer items are only ever activated, not selected

The original styles in list combined both to accommodate both drawer (activated) and menu (selected) contexts.